src/components/sessions/add-classes/AddClasses.tsx

```diff
@@ -0,0 +1,231 @@
+                    </Typography>
+                  </Box>
+                  <FormRow
+                    id={`classname-${classData.index}`}
+                    label="Name"
+                    questionType={QuestionType.TEXT}
+                    variant={FieldVariant.COMPACT}
+                  >
+                    <OutlinedInput
+                      autoComplete="new-password" // disable autocomplete
+                      className={classes.input}
+                      fullWidth
+                      id={`classname-${classData.index}`}
+                      placeholder="Class name"
+                      onChange={(e) =>
+                        onUpdateClass(i, { ...classData, name: e.target.value })
+                      }
+                      value={classData.name}
+                    />
+                  </FormRow>
+                  <DaysPicker
+                    days={classData.days}
+                    onChange={(days) => {
+                      onUpdateClass(i, { ...classData, days });
+                    }}
+                  />
+                  <FormRow
+                    id={`location-${classData.index}`}
+                    label="Location"
+                    questionType={QuestionType.TEXT}
+                    variant={FieldVariant.COMPACT}
+                  >
+                    <OutlinedInput
+                      autoComplete="new-password" // disable autocomplete
+                      className={classes.input}
+                      fullWidth
+                      id={`location-${classData.index}`}
+                      placeholder="Location"
+                      onChange={(e) =>
+                        onUpdateClass(i, {
+                          ...classData,
+                          location: e.target.value,
+                        })
+                      }
+                      value={classData.location}
+                    />
+                  </FormRow>
+                  <FormRow
+                    id={`class ${classData.index}`}
+                    label="Facilitator"
+                    questionType={QuestionType.SELECT}
+                    variant={FieldVariant.COMPACT}
+                  >
+                    <Select
+                      aria-label="Class facilitator"
+                      className={classes.input}
+                      displayEmpty
+                      fullWidth
+                      labelId={`class ${classData.index}`}
+                      onChange={(e) =>
+                        onUpdateClass(i, {
+                          ...classData,
+                          facilitator: e.target.value as number,
+                        })
+                      }
+                      value={classData.facilitator || ""}
+                      variant="outlined"
+                    >
+                      <MenuItem value="" className={classes.menuItem}>
+                        Facilitator
+                      </MenuItem>
```

Comment:
tiniest thing - could we add this colour style so it matches the placeholders for the other inputs?